clappr/clappr-android
# Media Control

### Goal

Create Clappr Media Control.

### How to Test
1. Run unit tests
2. Open Clappr Sample app
3. Click in the video
4. You should see the background getting dark with the user interaction in Media Control
